Extending/Content-Apps/index-vpre9.2.md

```diff
@@ -0,0 +1,354 @@
+---
+versionFrom: 9.0.0
+verified-against: 9.0.0
+state: complete
+updated-links: true
+meta.Title: "Content Apps"
+meta.Description: "A guide configuring content apps in Umbraco"
+---
+
+# Content Apps
+
+## What are Content Apps
+
+Content Apps are **companions** to the editing experience when working with content or media in the Umbraco backoffice.
+
+With Content Apps, editors can switch from editing 'Content' to accessing contextual information related to the item they are editing.
+
+![Content Apps in backoffice](images/content-app-1.png)
+
+### Default Content Apps
+
+**'Info'** - The 'Info' Content App is a default Content App for all items, displaying Links, History and Status of the current content item.
+
+### Custom Content Apps
+
+As an integrated part of Umbraco it is possible for you as a developer to create and provide your editors with helpful Content Apps.
+
+For example, you could create a Google Analytics integration within a Content App that displays to editors the current 'page views' for the content item they are editing.
+
+#### Controlling Appearance/Position
+
+You can associate an icon and control the position of your custom Content App. The position, for example between 'Content' and 'Info', is set via a 'weighting' number.
+
+#### Permissions
+
+Content Apps can be configured to appear dependent on Section, Content Type and User Group Permissions.
+
+#### Read-Only
+
+Content Apps are designed to be companions to the Content Item. They should enhance the editor's experience by enabling quick access to contextual information for the particular content item they are editing. Content Apps are not intended to be used for the editing content.
+
+## Creating a Custom Content App
+
+This guide explains how to set up a custom Content App in the following steps:
```

Comment:
```suggestion
This guide explains how to set up a Custom Content App in the following steps:
```
A little uppercase C for consistency 🙂

Reply:
@sophie-neale Thanks for your suggestion! 😄